stacker/providers/session_cache.py

```diff
@@ -0,0 +1,64 @@
+import json
```

Comment:
The providers directory is actually for service providers (like AWS - in theory we could have more in the future).  Might be worth giving this it's own module like `stacker/credential_cache.py`